<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="hSplitter">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QSplitter" name="vSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QTableView" name="tableView"/>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>分数段</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数学</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>语文</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>英语</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>平均分</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <item>
         <property name="text">
          <string>60分以下</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70分以下</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80分以下</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90分以下</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90分以上</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>平均分</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>最高分</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>最低分</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="bar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
        </attribute>
        <attribute name="title">
         <string>柱状图</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="barGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="barBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>刷新柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hSpacer1">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>451</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="barChart" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pie">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <attribute name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/images/pie2.png</normaloff>:/images/pie2.png</iconset>
        </attribute>
        <attribute name="title">
         <string>饼图</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="pieGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="pieLabel">
              <property name="text">
               <string>分析数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="pieDataComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>数学</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>语文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>平均分</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pieBtn">
              <property name="text">
               <string>刷新柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="holeLabel">
              <property name="text">
               <string>HoleSize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="holeDSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="stepType">
               <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pieSizelabel">
              <property name="text">
               <string>PieSize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="pieDSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pieThemeLabel">
              <property name="text">
               <string>主题</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="pieThemeComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>Light</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BlueCerulean</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BrownSand</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BlueNcs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HighContrast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bluelcy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="pieChart" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="stack">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/images/stacked.png</normaloff>:/images/stacked.png</iconset>
        </attribute>
        <attribute name="title">
         <string>堆叠图</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="stackGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="stackBtn">
              <property name="text">
               <string>刷新柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hSpacer3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>451</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="stackChart" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="percent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/images/percent.png</normaloff>:/images/percent.png</iconset>
        </attribute>
        <attribute name="title">
         <string>百分比图</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="percentGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="percentBtn">
              <property name="text">
               <string>刷新柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hSpacer4">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>451</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="percentChart" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="scatter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/images/scatter.png</normaloff>:/images/scatter.png</iconset>
        </attribute>
        <attribute name="title">
         <string>散点图</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="scatterGroupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="scatterBtn">
              <property name="text">
               <string>刷新柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hSpacer4_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>451</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="scatterChart" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="generateAction"/>
   <addaction name="statisticAction"/>
   <addaction name="quitAction"/>
  </widget>
  <action name="statisticAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/statistics.png</normaloff>:/images/statistics.png</iconset>
   </property>
   <property name="text">
    <string>统计</string>
   </property>
   <property name="toolTip">
    <string>数据统计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/out.png</normaloff>:/images/out.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="generateAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/thunder.png</normaloff>:/images/thunder.png</iconset>
   </property>
   <property name="text">
    <string>生成数据</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
